js/kucoinfutures.js

```diff
@@ -93,6 +93,7 @@ module.exports = class kucoinfutures extends kucoin {
                     'private': 'https://openapi-v2.kucoin.com',
                     'futuresPrivate': 'https://api-futures.kucoin.com',
                     'futuresPublic': 'https://api-futures.kucoin.com',
+                    'webFront': 'https://futures.kucoin.com/_api/web-front',
```

Comment:
@samgermain is this an internal API?

Reply:
@carlosmiei I really don't don't know, I found out about it from https://github.com/ccxt/ccxt/issues/16549 but that's all I know about it

Reply:
@samgermain I see but I'm pretty sure it is, @kroitor what should we do here?